test/test_utils/sagemaker.py

```diff
@@ -268,3 +267,15 @@ def execute_sagemaker_remote_tests(image):
         with context.prefix(f"source {tag}/bin/activate"):
             context.run("pip install -r requirements.txt", warn=True)
             context.run(pytest_command)
+
+
+def generate_empty_report(report):
+    """
+    Creating an empty reports for EIA and TF PY2 Inference Images
+    :param report: CodeBuild Report
+    Returns: None
+    """
+    test_cases = [TestCase('sagemaker-local', 'eia', 1, 'Skipped SM Local on EIA', '')]
```

Comment:
can we make this function more generalized? as in, can we have an argument for the names of the test cases?

Reply:
Made the changes in new revision.